.env-sample

```diff
@@ -80,4 +80,7 @@ COMMUNITY_CURRENCIES=20
 RELAYS='ws://localhost:7000,ws://localhost:8000,ws://localhost:9000'
 
 # Seconds to wait to allow disputes to be started
-DISPUTE_START_WINDOW=600
\ No newline at end of file
+DISPUTE_START_WINDOW=600
+
+# Number of notification messages sent to the admin, informing them of lack of solvers before disabling the community. The admin receives `MAX_MESSAGES - 1` notification messages.
+MAX_MESSAGES=10
```

Comment:
@Anyitechs I would recommend renaming this to have a much clearer name; right now "messages" sounds too generic, how about MAX_WARNINGS_TO_ADMIN_BEFORE_COMMUNITY_DEACTIVATION